### ISSUE TITLE
GET Suffix 'STATE' not found on object PART after docking.

### ISSUE THREAD

**Issue by fellipec** [xeger/kos-ramp#20]
After docking, sometimes, get error like this, when the control part changes and have not the `STATE` suffix. 
Needs to add a check to prevent this. Maybe not the only place such can happens.

```
           VERBOSE DESCRIPTION
GET Suffix 'STATE' not found on object PART(crewpod-habitation-25 (MIR),uid=2271379456)
An attempt was made to get a suffix called:
    state
from an object of type:
    PartValue
when that object does not have that
suffix usable in that way.
Possible causes are:
  - No PartValue ever has a state suffix.
  - There is such a suffix, but it can't be used
      with a get operation.
  - There is such a suffix on some instances
      of PartValue, but not this particular one.
__________________________________________
__________________________________________
At 1:/lib_dock.ks, line 218
  if port:state = "PreAttached" {
          ^
Called from 1:/lib_dock.ks, line 114
    if not dockPending(ship:controlpart) {
                            ^
Called from 1:/dock.ks, line 72
        dockApproach(dockD, dockV).
                            ^
Called from 1:/rendezvous, line 58
run dock.
    ^
```

**Comment by xeger** [comment]
hmm; there's probably some method we can call on the part to check if it's a port.

**Comment by fellipec** [comment]
Yes, that was my first idea, check if there is moduledockingport. 

But the function that throws the error receive a part via a parameter, and that parameter was made sure it was a docking port beforehand. My guess is that when the craft dock someway messes with indexes of parts lists or something like that and makes kOS change that part by itself.

I'll do more research on this. Anyway, those bugs in docking aren't that severe because they happen when we dock successfully, thus fulfilling the purpose of the program. 😃

**Comment by fellipec** [comment]
Fixed in the last commit on #19
